addons/html_builder/static/src/website_builder_action.js

```diff
@@ -97,10 +125,90 @@ class WebsiteBuilder extends Component {
             targetDoc: this.websiteContent.el.contentDocument,
         });
         this.websiteService.pageDocument = this.websiteContent.el.contentDocument;
+        if (this.translation) {
```

Comment:
Is it possible that `this.translation` is not set when arriving here: in other world, is it possible that `onIframeLoad` is called before `onPatched` ?